test/unit-tests/mobile/devices-service.ts

```diff
@@ -437,6 +440,8 @@ describe("devicesService", () => {
 			it("does not fail when androidDeviceDiscovery startLookingForDevices fails", async () => {
 				(<any>androidDeviceDiscovery).startLookingForDevices = (): Promise<void> => { throw new Error("my error"); };
 				iOSDeviceDiscovery.emit("deviceFound", iOSDevice);
+				let hostInfo = testInjector.resolve("hostInfo");
+				hostInfo.isDarwin = true;
```

Comment:
I planed to change this test. Will update it with correct message, in appliance with refactoring.